include/net/ethernet.h

```diff
@@ -43,15 +47,54 @@ extern "C" {
 
 #define NET_ETH_MINIMAL_FRAME_SIZE	60
 
-enum eth_hw_caps {
+enum ethernet_hw_caps {
```

Comment:
A bit longish prefix, I liked the shorter one more but can live with this one too.